cider/tests/functionality/CiderCaseWhenTest.cpp

```diff
@@ -272,6 +354,93 @@ CASE_WHEN_AGG_TEST(CiderCaseWhenSequenceTestBase, caseWhenAggTest);
 CASE_WHEN_AGG_TEST(CiderCaseWhenSequenceWithNullTestBase, caseWhenAggTest);
 CASE_WHEN_AGG_TEST(CiderCaseWhenRandomWithNullTestBase, caseWhenAggTest);
 
+#define CASE_WHEN_ARROW_TEST(TEST_CLASS, UNIT_NAME)                                      \
+  TEST_F(TEST_CLASS, UNIT_NAME) {                                                        \
+    assertQueryArrow(                                                                    \
+        "SELECT col_int, CASE WHEN col_int > 3 THEN 1 ELSE 0 END FROM test");            \
+    assertQueryArrow(                                                                    \
+        "SELECT col_double, CASE WHEN col_int > 5 THEN 4 ELSE 3"                         \
+        " END FROM test");                                                               \
+    assertQueryArrowIgnoreOrder(                                                         \
+        "SELECT col_int, CASE WHEN col_int > 30 THEN 1 ELSE 0 END FROM test");           \
+    GTEST_SKIP_("FIXME(haiwei): Case when with null value");                             \
```

Comment:
I remember one `GTEST_SKIP_` will disable all later case?

Reply:
Only disable the TEST_F's case. Case when with null value will been finished in another sub-task.